### DISCUSSION TITLE
generate stats for job

### DISCUSSION THREAD

**Discussion by oliviermollier** [jhuckaby/Cronicle#540] [Q&A] [answered]
Hello, I use cronical for shell script execution and I'm interested in the feature to have detailed metrics. I found the i should write in the stdout the values in json  like : { perf: { db: 18.51, http: 3.22, gzip: 0.84 } }

But when i write this value it doesn't work. 

Can someone already implemented it?

Can you share a sample script shell that will make such stats?

Should it be the last line ?

Thx a lot

**Comment by oliviermollier** [comment, marked as answer]
it's ok i found myself:
echo {"perf":{"1:pspur01p":3,"2:pspur01p":4}}

**Comment by jpdom** [comment]
@jhuckaby  having the same problem.
This is the output from my shell command:
![image](https://user-images.githubusercontent.com/16657832/195136976-6a32ba27-fb1c-4529-b7bb-267f4adf1a3d.png)

And yet it doesn't generate stats.

I have "Interpret JSON in output" enabled btw.

**Comment by jhuckaby** [comment]
@John1997x Syntax error in your JSON.  Trailing comma.

**Reply by jpdom** [reply]
Thank you !! Looked so many times and didn't see that.
